packages/rum/src/rum.ts

```diff
@@ -359,6 +360,7 @@ function trackView(lifeCycle: LifeCycle, handler: (startTime: number, event: Rum
         loadingTime: view.loadingTime ? msToNs(view.loadingTime) : undefined,
         loadingType: view.loadingType,
         measures: view.measures,
+        url: view.location.href,
```

Comment:
With this approach, url from ViewContext is still wrong so it can still be wrongly associated to other event types.
What about fixing it there?

Reply:
I wrote a new fix, let me know if it looks right :)